src-ui/src/app/services/open-documents.service.ts

```diff
@@ -53,6 +57,11 @@ export class OpenDocumentsService {
     }
   }
 
+  setDirty(documentId: number, dirty: boolean) {
+    if (dirty) this.dirtyDocuments.add(documentId)
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/jonaswinkler/paperless-ng/pullRequest?prid=7592507)
```suggestion
    if (dirty) {this.dirtyDocuments.add(documentId)}
```